msgraph/core/_constants.py

```diff
@@ -0,0 +1,11 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+"""
+Application constants
+"""
+REQUEST_TIMEOUT = 100
+CONNECTION_TIMEOUT = 30
+BASE_URL = 'https://graph.microsoft.com/'
```

Comment:
Note (no action at this time): we will want to support national clouds
https://docs.microsoft.com/en-us/graph/deployments?view=graph-rest-1.0

Reply:
Let's rename this DEFAULT_BASE_URL, and then add a comment that this value can be set when creating a client via the HttpClientFactory and GraphClient.